releasenotes/notes/upgrade-tiktoken-removed-a10db578f5354615.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    openai: ``tiktoken`` has been removed from ``ddtrace`` as a dependency, and streamed chat/completions will default to having their token counts estimated
```

Comment:
probably don't need to reference the removal of tiktoken here, just focusing on the product changes only.

minot nit, not a big deal.